clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -312,8 +312,7 @@ public void onFailure(RuntimeException e) {
                 protocolType(),
                 metadata());
 
-        // create the request for the coordinator
-        log.debug("Issuing request ({}: {}) to coordinator {}", ApiKeys.JOIN_GROUP, request, this.coordinator.id());
+        log.debug("Sending JoinGroup ({}) to coordinator {}", request, this.coordinator);
```

Comment:
It currently looks like "Node(id, host, port)," which doesn't seem too bad to me. What do you think?